src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
-spring.data.rest.base-path=/api
+spring.data.rest.basePath=/api
```

Comment:
Both of these will work, but `basePath` is the version that's documented.

http://docs.spring.io/spring-data/rest/docs/current/reference/html/#_changing_the_base_uri